radix-substate-store-queries/src/typed_substate_layout.rs

```diff
@@ -153,7 +153,7 @@ pub enum TypedMainModuleSubstateKey {
     NonFungibleVault(NonFungibleVaultTypedSubstateKey),
     ConsensusManager(ConsensusManagerTypedSubstateKey),
     ValidatorField(ValidatorTypedSubstateKey),
-    AccessController(AccessControllerTypedSubstateKey),
+    AccessController(AccessControllerV2TypedSubstateKey),
```

Comment:
If v1 and v2 are the same, we should keep `AccessControllerTypedSubstateKey`? or we should have `AccessControllerV1` and `AccessControllerV2`?

Reply:
I think that this is similar to the other discussion, I believe that blueprint updates that extend the state (adding fields as an example) we can use latest versions.